packages/@ourworldindata/grapher/src/schema/grapher-schema.003.yaml

```diff
@@ -260,9 +260,12 @@ properties:
                           - latest
             variableId:
                 type: integer
-                description: "Variable ID to show. TODO: remove this and use dimensions instead"
+                description: "Variable ID to show. TODO: remove this once we have a convention of using the first y dimension instead"
```

Comment:
since this is yaml, we could also just be using yaml comments for TODOs?

Reply:
Very good point